app/views/questions/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="question page">
+
+	<%=render 'questions/question' %>
+  <% @question.comments.each do |comment| %>
+    <% @old_comment = comment %>
+    <%=render 'comments/show'%>
+  <%end%>
+  <%=render 'comments/new_q_comment'%>
+  <% @question.answers.each do |answer|%>
+    <% @old_answer = answer%>
+    <%=render 'answers/show'%>
+    <% answer.comments.each do |comment|%>
+      <% @old_comment = comment %>
+      <%=render 'comments/show'%>
```

Comment:
Getting intense with your partials :+1:.

The protip here is that you really basically never ever want to be setting variables in the view. Any logic that pops up in here should be moved immediately to the controller or to the model. In this case, though, you're just setting variables to be used in the partials. There is actually some built in logic for this. Check out http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials and search for "locals".